websites/A/AnimeDrive/presence.ts

```diff
@@ -0,0 +1,87 @@
+const presence = new Presence({
+		clientId: "1211027222815711282",
+	}),
+	browsingTimestamp = Math.floor(Date.now() / 1000);
+
+presence.on("UpdateData", async () => {
+	const presenceData: PresenceData = {
+			largeImageKey:
+				"https://cdn.discordapp.com/icons/702098578251841577/be50b5dbcb30bd99512b6df29284db0c.webp",
+			startTimestamp: browsingTimestamp,
+			buttons: [
+				{
+					label: "Meglátogatás",
+					url: "https://animedrive.hu/",
+				},
+			],
```

Comment:
According to current [guidelines](https://github.com/PreMiD/Presences/blob/main/.github/CONTRIBUTING.md#presencets), buttons that redirect to main pages are not allowed.

Reply:
I will be only able to fix this later I will push in about 2 hours.

Reply:
I'm not at home I could only do the Icon on phone